modular_skyrat/modules/colony_fabricator/code/design_datums/flatpack_machines.dm

```diff
@@ -132,7 +132,7 @@
 // PACMAN generator but epic!!
 
 /datum/design/flatpack_solids_generator
-	name = "Flatpacked Solid Fuel Generator"
+	name = "Flatpacked S.O.F.I.E. Solid Fuel Generator"
```

Comment:
I don't think P.A.C.M.A.N.s have "plasma" in their names, right? It's basically the same fuel as P.A.C.M.A.N.s anyway, so I don't think there'll be too much confusion here.
```suggestion
	name = "Flat-Packed S.O.F.I.E. Generator"
```